### ISSUE TITLE
NeovimStreamApi#sendAtomic wrongly wraps requestArgs into List

### ISSUE THREAD

**Issue by renierG** [esensar/neovim-java#139]
The Stream converting the `requestMessages` in `NeovimStreamApi#sendAtomic` is wrongly wrapped into an additional `List.of(...)`.

Causes `RpcException{rpcError=RpcError{type=Validation, message='Items in calls array must be arrays of size 2'}}` on usage.

```java
package com.ensarsarajcic.neovim.java.api; 

import ...

@NeovimApiClient(name = "full_stream_api", target = 9)
public final class NeovimStreamApi extends BaseStreamApi implements NeovimApi {

    ...

    @Override
    public CompletableFuture<AtomicCallResponse> sendAtomic(List<RequestMessage> requestMessages) {
        var requestArgs = List.of(
                requestMessages.stream()
                        .map(
                                requestMessage -> List.of(requestMessage.getMethod(), requestMessage.getArguments())
                        ).collect(Collectors.toList())
        );
        return sendWithResponseOfType(
                new RequestMessage.Builder(CALL_ATOMIC)
                        .addArgument(requestArgs),
                AtomicCallResponse.class
        );
    }

    ...
}
```

Possible Fix:
```java 
...
var requestArgs = requestMessages.stream()
        .map(requestMessage -> List.of(requestMessage.getMethod(), requestMessage.getArguments()))
        .toList();
...
```

**Comment by esensar** [comment]
Thank you for reporting this. Nice find!

Would you like to open a PR with that suggested fix? If not, I will take care of it when I find some time.

**Comment by esensar** [comment]
Released in 0.5.0
